<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>停车场收费系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLicensePlateNumber">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>351</width>
      <height>67</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>6</number>
    </property>
    <property name="text">
     <string>京A88888</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxCarPhoto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>450</width>
      <height>370</height>
     </rect>
    </property>
    <property name="title">
     <string>车辆照片</string>
    </property>
    <widget class="QLabel" name="labelCarPhote">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>30</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string>车辆照片</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGateAnimation">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>450</width>
      <height>370</height>
     </rect>
    </property>
    <property name="title">
     <string>闸机动画</string>
    </property>
    <widget class="QLabel" name="labelGateAnimation">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>30</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string>闸机动画</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnter">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>入库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLeave">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>出库</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>471</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>共计停放0天2时30分，共计收费3元</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSimulateEnter"/>
    <addaction name="actionSimulateLeave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionChangeStandard"/>
    <addaction name="actionAutoBarrierControl"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="actionQueryStatistics"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuQuery"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionChangeStandard">
   <property name="text">
    <string>收费标准</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionAutoBarrierControl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动开闸</string>
   </property>
  </action>
  <action name="actionQueryStatistics">
   <property name="text">
    <string>查询统计</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSimulateEnter">
   <property name="text">
    <string>模拟进入</string>
   </property>
  </action>
  <action name="actionSimulateLeave">
   <property name="text">
    <string>模拟离开</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
